Src/DependencyCollector/NuGet/ApplicationInsights.config.install.xdt

```diff
@@ -17,6 +17,18 @@
         <Add>localhost</Add>
         <Add>127.0.0.1</Add>
       </ExcludeComponentCorrelationHttpHeadersOnDomains>
+      <ExcludeDiagnosticSourceActivities>
```

Comment:
please also take care about ASP.NET Core DiagnosticListener.

Reply:
Do you mean ```Microsoft.AspNet.TelemetryCorrelation```? It's already excluded

Reply:
I mean  https://github.com/Microsoft/ApplicationInsights-aspnetcore/blob/develop/src/Microsoft.ApplicationInsights.AspNetCore/DiagnosticListeners/Implementation/HostingDiagnosticListener.cs

DependencyCollector package will be also used in AspNet Core SDK. 
AspNet Core SDK does not have xml configuration file, so `Microsoft.AspNetCore` should be excluded in some other way.